pyanaconda/modules/payload/sources/initialization.py

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright (C) 2019 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+import os
+import stat
+
+from pyanaconda.payload.utils import mount, unmount
+from pyanaconda.modules.common.constants.objects import DEVICE_TREE
+from pyanaconda.modules.common.constants.services import STORAGE
+from pyanaconda.modules.common.errors.payload import SourceSetupError
+from pyanaconda.modules.common.structures.storage import DeviceData
+from pyanaconda.modules.common.task import Task
+
+
+class TearDownInstallationSourceTask(Task):
```

Comment:
We should probably figure out namespacing (per-source type folders) or maybe better per source-type class naming (including LiveOS in class name) at some point, but this can be maybe done when more sources are added.
Nothing indicates this is LiveOS specific task (I assume it is).

Reply:
Good point. We will look on that in the future commits.